crytic_compile/platform/foundry.py

```diff
@@ -118,68 +118,62 @@ def is_supported(target: str, **kwargs: str) -> bool:
         return os.path.isfile(os.path.join(target, "foundry.toml"))
 
     @staticmethod
-    def config(working_dir: str) -> Optional[PlatformConfig]:
+    def config(target: str) -> Optional[PlatformConfig]:
         """Return configuration data that should be passed to solc, such as remappings.
 
         Args:
-            working_dir (str): path to the working directory
+            target (str): path to the target
 
         Returns:
             Optional[PlatformConfig]: Platform configuration data such as optimization, remappings...
         """
         result = PlatformConfig()
-        result.remappings = (
-            subprocess.run(["forge", "remappings"], stdout=subprocess.PIPE, check=True)
-            .stdout.decode("utf-8")
-            .replace("\n", " ")
-            .strip()
+        LOGGER.info("'forge config --json' running")
+        json_config = json.loads(
+            subprocess.run(["forge", "config", "--json"], stdout=subprocess.PIPE, check=True).stdout
```

Comment:
On a second look, I think target is always a directory here, so scratch that last part of my comment.

Unrelated to this PR, but I think the check [here](https://github.com/crytic/crytic-compile/pull/488/files#diff-6f65aa5b97bfca7b44a766ca4c355963fd9442ee6f46d72e8739985ee1bf06e4R131-R139) may be a bit off, wouldn't it make more sense to look for the foundry/etc project in the path of the file rather than in cwd? e.g. if the file is /a/b/c/d/contracts/foo.sol, I'd check to see if any of [/a/b/c/d/contracts, /a/b/c/d, /a/b/c, /a/b, /a, /] is a project. Most users are probably going to be running the tool with cwd=the project folder, so it may work as-is now, but it doesn't feel completely correct.